core/src/gl/uniformEntries.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <string>
+
+namespace Tangram {
+
+namespace UniformEntries {
```

Comment:
Do you think we can simplify some of these names? e.g. `namespace Uniform { ...`, `using Id/Index/Handle = uint16_t; `, and we may not need `UniformEntry` at all